firmware/application/apps/pocsag_app.cpp

```diff
@@ -195,26 +174,17 @@ void POCSAGAppView::on_packet(const POCSAGPacketMessage* message) {
                 console.write(pocsag_state.output);
             }
 
-            if (logger && logging())
+            /*if (logger && logging())
```

Comment:
Double check this isn't commented out

Reply:
Don't know what to say here ^^

Reply:
:) I hate the raw logging so I always comment it out for my personal builds. I plan to make it configurable at some point.
Pocsag is my favorite app. I can't want to start fixing it up.